.travis.yml

```diff
@@ -42,24 +42,19 @@ matrix:
     - env: DISTRIB="conda" PYTHON_VERSION="3.6.2" INSTALL_MKL="true"
            NUMPY_VERSION="1.13.1" SCIPY_VERSION="0.19.1" PANDAS_VERSION="0.20.3"
            CYTHON_VERSION="0.26.1" COVERAGE=true
-      if: type != cron
-    # This environment use pytest to run the tests. It uses the newest
-    # supported Anaconda release (5.0.0). It also runs tests requiring Pandas.
-    - env: USE_PYTEST="true" DISTRIB="conda" PYTHON_VERSION="3.6.2"
-           INSTALL_MKL="true" NUMPY_VERSION="1.13.1" SCIPY_VERSION="0.19.1"
-           PANDAS_VERSION="0.20.3" CYTHON_VERSION="0.26.1"
-           TEST_DOCSTRINGS="true"
+           CHECK_PYTEST_SOFT_DEPENDENCY="true"
```

Comment:
We could also put this in its own worker to be sure it's sandboxed, but I guess it's okay as it is.

Reply:
> We could also put this in its own worker to be sure it's sandboxed, but I guess it's okay as it is.

I thought about too but I thought this is a bit of a waste to have another build and run tests only for 5 seconds.